packages/suite/src/views/wallet/transactions/components/TransactionSummary.tsx

```diff
@@ -137,14 +138,16 @@ export const TransactionSummary = ({ account }: TransactionSummaryProps) => {
                     </Column>
                 </>
             )}
-            <SummaryCards
-                selectedRange={selectedRange}
-                dataInterval={dataInterval}
-                data={data}
-                localCurrency={localCurrency}
-                account={account}
-                isLoading={isLoading}
-            />
+            {!areNetworkTransactionsUnreliable(account.symbol) && (
```

Comment:
Are we really sure about this?

Reply:
Jakub's reasoning was: If we can't tell reliably that the summary info here is correct, hide it.It would make sense in case when we completely hide the phishing transactions. 

And since we can't tell forehand how many of such transactions are there, we hid it. Personally, I'd keept it. 

With you suggested change of adding another filter that would hide / display visible transaction, it would be better imho.